lib/bundler/source/git.rb

```diff
@@ -50,12 +50,7 @@ def hash
       end
 
       def eql?(o)
-        o.is_a?(Git)         &&
-        uri == o.uri         &&
-        ref == o.ref         &&
-        branch == o.branch   &&
-        name == o.name       &&
-        version == o.version &&
+        o.is_a?(Git) && uri == o.uri && ref == o.ref && branch == o.branch && name == o.name && version == o.version &&
```

Comment:
Me neither, but unless you do it, rubocop will fail. I can investigate if it's possible to rubocop ignore just this part of code, but maybe it's not a good idea.